packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -54,14 +54,16 @@ export const MediaGallery = (props: MediaGalleryProps): JSX.Element => {
         remoteVideoViewOptions={remoteVideoViewOptions}
         styles={VideoGalleryStyles}
         layout="floatingLocalVideo"
+        /* @conditional-compile-remove-from(stable) */
```

Comment:
Please add a tag to find it easier to find (and eliminate) this conditional compilation tag later.